relayer/chain/parachain/writer.go

```diff
@@ -120,6 +121,8 @@ func (wr *ParachainWriter) WriteToParachainAndWatch(ctx context.Context, extrins
 				return fmt.Errorf("parachain write status was dropped, invalid, usurped or finality timed out")
 			}
 			if status.IsFinalized {
+				log.WithFields(log.Fields{
+					"extrinsic": extrinsicName, "block": status.AsFinalized}).Info("extrinsic finalized")
```

Comment:
Thinking this can be a debug log, but not major. 😄

Reply:
Yes, just for more debug and the `blockHash` added here will be useful to search the block and extrinsic which error happens.